password-access.qmd

```diff
@@ -2,6 +2,8 @@
 title: "Password-based Access"
 ---
 
+This page is written primarily for workshop leads (mentors and other partners) who are inviting people to the Hub, as a collaborator for developing workshop materials or as a participant.
```

Comment:
I think in the password-based access page, this isn't quite right. Maybe something like: 
```suggestion
This page is written primarily for workshop leads (mentors and other partners) who will be inviting participants to sign in to the hub on the day of the workshop, without using GitHub for authentication.
```